### DISCUSSION TITLE
Multiple Bookmark Lists

### DISCUSSION THREAD

**Discussion by gfreundt** [codewars/codewars.com#2729] [Ideas]
Hello everyone,
I've been bookmarking many katas, and I would like the opportunity to categorize my bookmarks creating multiple bookmark lists (like a browser). I can think of a few categories for me:

- Solved katas that I would like to keep bookmarked (for many different reasons).
- Katas in process of being solved.
- Katas "solved" on my IDE but can't complete because they time out (happens a lot to me!)
- Katas untouched for future work.

I am sure everyone has their own needs and categories.

My suggestion: allow users to create multiple bookmark lists with personalized names.

Thanks,

Gabriel

**Comment by kazk** [comment]
I guess we can add an option to make collections private.

**Reply by gfreundt** [reply]
sounds like a good solution.
